packages/components-native/src/InputCurrency/InputCurrency.test.tsx

```diff
@@ -82,8 +82,6 @@ describe.each([{ includeATLContext: true }, { includeATLContext: false }])(
       });
 
       fireEvent.changeText(getByLabelText(placeHolder), `${value}`);
-      fireEvent(getByLabelText("Price"), "blur");
-      fireEvent(getByLabelText("Price"), "blur");
```

Comment:
Note to reviewer - this `blur` triggers [this callback](https://github.com/GetJobber/atlantis/blob/fc326c01fb124414c9a84198655d88476c40e7ae/packages/components-native/src/InputCurrency/InputCurrency.tsx#L193-L201). The onBlur callback references the `field` value from react-hook-form, though the test setup doesn't wrap the component in a form controller/provider. This will cause the input to render a display value of `0`.

Simple fix is to avoid triggering blur events in specs, but also happy to reconsider alternative solutions!